New version: ORToolsGenerated v0.0.2

JuliaRegistrator opened the pull request:
- Registering package: ORToolsGenerated
- Repository: https://github.com/google/or-tools
- Created by: @dourouc05
- Version: v0.0.2
- Commit: 78bc950b494b3c906ceed88253497cef7d134ca1
- Reviewed by: @dourouc05
- Reference: https://github.com/google/or-tools/commit/78bc950b494b3c906ceed88253497cef7d134ca1#commitcomment-162533863
- Description: Google's Operations Research tools:
- Release notes:
<!-- BEGIN RELEASE NOTES -->
`````
breaking changes / changelog: continued development; this should be the first useable version
`````
<!-- END RELEASE NOTES -->

<!-- ae9404edfd72f1de901db3e818bff6d68e1033f2a40e4f25e4af6c579aabbd89dfa5a40efacf6336f6f89800aaa8d234c46f2d0ac3b9eb8d004644ae89b40bb44f4ab3dd3840c73d55c63753fb1a696894f324d0874a444ea5399b5fb337b55f18f15e11898d143e77dad0bdc9d90924f7ffb828b942f7fc86c084ec9120aecd751f2528ee55c47b5141946c4c4edf0e87f15d1ff273552d3a912588b161e0497cd43c98d90a3b7ad899afe0bcdc62cbcd9372612e3545c02008e309f447fbe73949c158eb4c5fc762b9c4a4d4947e5cd266bee6e129f954e84c57b814906043c46494c3a86cbc12ff0a0db6f55e02f60a626d95ccbb1bd6c25be27f13ab4fdf -->

dourouc05 commented:
I'm not sure the failure is related to this PR, as the error messages show HTTP timeouts :/. 

(Moreover, this PR is a blocker for https://github.com/JuliaRegistries/General/pull/135129.)

JuliaRegistrator committed:
New version: ORToolsGenerated v0.0.2

UUID: 6b269722-41d3-11ee-be56-0242ac120002
Repo: https://github.com/google/or-tools.git
Tree: 0f90667c4eadc196191fae0cbbe8159ddc19ade2

Registrator tree SHA: 50f504d641745716a5b3eabaf681d3a4937d2ae3
